master-params.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file was added by accident?

Reply:
removed in https://github.com/elastic/rally/pull/830/commits/947c11016ee26f62abea2d84f686bee9eaebc3d2